Searching & sorting/Binary Search.py

```diff
@@ -0,0 +1,15 @@
+def binary_search(array, search):
+    low = 0
+    high = len(array) -1
+
```

Comment:
add a comment for explanation  and try to make a different folder inside the searching & sorting folder